src/components/get_started/Contribute.jsx

```diff
@@ -53,9 +56,9 @@ class Contribute extends Component {
     markdown: '',
   };
   componentDidMount() {
-    const mdPath = require('./contribute.md');
-
-    fetch(mdPath)
+    fetch(constants.CONTRIBUTE, {
```

Comment:
This does not work! 
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://scigym.s3.eu-central-1.amazonaws.com/markdown/contribute.md. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).